cnf-certification-test/webserver/index.html

```diff
@@ -9,6 +10,9 @@
           sizes="any"
           href="https://ux.redhat.com/assets/logo-red-hat.svg">
     <link rel="stylesheet" href="https://ux.redhat.com/assets/packages/@rhds/elements/elements/rh-footer/rh-footer-lightdom.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```

Comment:
JQuery [v3.7.1](https://github.com/jquery/jquery/releases/tag/3.7.1) is the latest release.  Should we be using that?